### DISCUSSION TITLE
strange error due to Chinese fonts missing

### DISCUSSION THREAD

**Discussion by willin** [community/community#10808] [Actions]
local dev:
![image](https://user-images.githubusercontent.com/1890238/151779657-93549341-29fd-4bf4-9ca4-dff8426cc92c.png)

github actions generated:

![image](https://user-images.githubusercontent.com/1890238/151779728-7b32510c-5fb9-4875-9f99-f9bbff6fe344.png)

source at: https://github.com/willin/willin.wang/blob/main/scripts/mdx.mjs#L85

**Comment by ethomson** [comment]
Hi @willin - sorry that you're seeing some trouble.  Are you able to install the fonts?  You have administrator access on the runners and can install any software package that's missing.
